app/cli/internal/action/attestation_push.go

```diff
@@ -160,6 +160,19 @@ func (action *AttestationPush) Run(ctx context.Context, attestationID string, ru
 		return nil, err
 	}
 
+	// execute policy evaluations
+	// We do not want to evaluate policies here during render since we want to do it in a separate step
+	statement, err := renderer.RenderStatement(ctx)
```

Comment:
I thought about also running it on att.add but I am not sure it's needed yet.

Reply:
Probably not, since we want to block only on "push"